src/App.js

```diff
@@ -30,20 +30,21 @@ const { Content, Footer, Sider } = Layout;
 firebase.initializeApp(firebaseConfig)
 
 export const AuthContext = createContext(undefined)
+export const UserContext = createContext(undefined)
```

Comment:
you created UserContext, but then...

Reply:
Ah, I see my mistake, I thought I could pass it through with AuthContext.Provider. I'll add another Provider